### PR TITLE
reload after POST submissions

### PR DESCRIPTION
**Pull request by ameotoko**
This PR implements the Contao way to handle POST forms submissions (and an overall good practice):

- page is reloaded after POST request
- flash messages are used

This makes sure that user can not spam POST requests by pressing "refresh" button.

### CODE REVIEW

File: src/Module/TokenLogin.php [hunk 1]
```diff
@@ -147,12 +148,13 @@ protected function getResponse(Template $template, ModuleModel $model, Request $
...
-                    $template->message       = $this->translate('MSC.token_login.form_success');
+                    Message::addInfo($this->translate('MSC.token_login.form_success'), 'richardhj_token_login');
                 } else {
-                    $template->hasError = true;
-                    $template->message  = $this->translate('MSC.token_login.form_error');
+                    Message::addError($this->translate('MSC.token_login.form_error'), 'richardhj_token_login');
                 }
             }
+
+            Controller::reload();
```
Comment (richardhj) [inline src/Module/TokenLogin.php:157]:
```suggestion
            return new Response($request->getUri());
```

**Comment by ameotoko** [comment]
The large amount of changes in the template file is due changed indentation, because the form is inside a separate `if` statement now...

The only real changes are – the message block is rendered only once and regardless of whether or not the form itself is shown.

File: src/Module/TokenLogin.php (hunk 1 above)
Comment (ameotoko) [reply src/Module/TokenLogin.php:157]: I'm a fan of Contao helpers, where it makes code a little bit cleaner😅 . I mean it does the same thing eventually, but with some extra things taken care of under the hood, which is nice.

Should I still change it?

Comment (ameotoko) [reply src/Module/TokenLogin.php]: fixed in d1d15e3

**Pull request merged**